libtransmission/file.h

```diff
@@ -145,6 +145,20 @@ tr_sys_path_info;
 
 /* Path-related wrappers */
 
+/**
+ * @brief Portability wrapper for various in-kernel file copy functions, with a
+ *        fallback to a userspace read/write loop.
+ *
+ * @param[in]  src_path  Path to source file or directory.
+ * @param[in]  dst_path  Path to destination file or directory.
```

Comment:
The function doesn't support copying directories as it is coded right now, so let's drop the "or directory" part. Taking it one step further, you may even consider renaming it to `tr_sys_file_copy()` to better relay the intent.